add variable item `tidb_track_aggregate_memory_usage` (#4508)
pingcap/docs: system-variables.md

- Default value: 0.8
- TiDB triggers an alarm when the percentage of the memory it takes exceeds a certain threshold. For the detailed usage description of this feature, see [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-new-in-v409).
- You can set the initial value of this variable by configuring [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-new-in-v409).

> ### `tidb_track_aggregate_memory_usage` <!-- New in v5.0.0-rc -->

Reviewer: ```suggestion
### `tidb_track_aggregate_memory_usage` <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
```

> ### `tidb_track_aggregate_memory_usage` <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>

Reviewer: ```suggestion
### `tidb_track_aggregate_memory_usage` <span class="version-mark">New in v5.0.0-rc</span>
```

> **Warning:**
>
> `tidb_track_aggregate_memory_usage` is currently an experimental feature. It is not recommended to use this feature in the production environment.

> > 

Reviewer 2: ```suggestion

```

- Scope: SESSION | GLOBAL
- Default value: OFF

> - This variable indicates whether to track the memory usage of the aggregate function. When you enable this feature, it counts the memory usage of the aggregate function, which might cause the overall SQL memory statistics to exceed the threshold [`mem-quota-query`](/tidb-configuration-file.md#mem-quota-query), and then be affected by the behavior defined by [`oom-action`](/tidb-configuration-file.md#oom-action).

Reviewer 2: ```suggestion
- This variable controls whether to track the memory usage of the aggregate function. When you enable this feature, TiDB counts the memory usage of the aggregate function, which might cause the overall SQL memory statistics to exceed the threshold [`mem-quota-query`](/tidb-configuration-file.md#mem-quota-query), and then be affected by the behavior defined by [`oom-action`](/tidb-configuration-file.md#oom-action).
```
